test/browser/question.test.js

```diff
@@ -46,4 +52,23 @@ describe('Question page', () => {
   it('displays question answer box', async() => {
     expect(await questionPage.getElement('#question-field')).to.not.be.null;
   });
+
+  it('displays an error message in the summary when you try to save an empty answer', async() => {
+    await questionPage.saveAnswer('');
+    expect(await questionPage.getElementText('.govuk-error-summary'))
+      .contain(i18n.question.textareaField.error.empty);
+  });
+
+  it('displays an error message above the field when you try to save an empty answer', async() => {
+    await questionPage.saveAnswer('');
+    expect(await questionPage.getElementText('#question-field-error'))
+      .contain(i18n.question.textareaField.error.empty);
+  });
+
+  if (testingLocalhost) {
```

Comment:
I'm guessing this a temporary measure until the backend work is done? I'd rather avoid writing tests that can't be run against preview and AAT.

Reply:
Yeah this is just until the api is ready